taccsite_cms/default_secrets.py

```diff
@@ -1,305 +0,0 @@
-# TACC CMS SITE TEMPLATE SETTINGS.
```

Comment:
do we need this to be deleted?

Reply:
I think not.

I propose it be cleaned up as multiple `...example.py` files, like Portal has. [A `settings_example.py` file specifically would resolve my blog concern (because I can document how to add it).](https://github.com/TACC/Core-CMS/pull/341#discussion_r708533776)

Reply:
There is useful documentation in this file, like

- [CMS_TEMPLATES](https://github.com/TACC/Core-CMS/blob/1248117/taccsite_cms/default_secrets.py#L59-L94)
- [FEATURES](https://github.com/TACC/Core-CMS/blob/1248117/taccsite_cms/default_secrets.py#L123-L157)
- [BRANDING & LOGOS & FAVICON](https://github.com/TACC/Core-CMS/blob/1248117/taccsite_cms/default_secrets.py#L167-L237)

I am not saying it must be here, but documentation should not be deleted without replacement or explanation that we agree makes it moot.

Reply:
- This file has been recently added (374a279) as `taccsite_cms/settings_custom.example.py`.
- I have created a new issue to revisit documentation after this PR: #343.